### PR TITLE
[25230] Update Nextcloud themeing

### PR DESCRIPTION
**Pull request by acomdigital**

### CODE REVIEW

File: docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js.old
Comment (col-panic) [inline docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js.old:1]: @acomdigital what is the meaning of this file? Is this being actively referenced?

File: docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js [hunk 1]
```diff
@@ -273,69 +275,295 @@ const elxsNavigation = {
     },
     loadConfig() {
         const url = "/.eenv.properties.json";
-        return fetch(url).then((resp) => resp.json())
-            .then((data) => data)
-            .catch((error3) => {
+        return fetch(url).then((resp) => resp.json()).then((data) => data).catch((error3) => {
                 return {
                     "ENABLE_BOOKSTACK": true,
```
Comment (col-panic) [inline docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js:280]: Does this mean that this is the default setting if not `/.eenv.properties.json` can be found?

File: docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js.old
Comment (acomdigital) [reply docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js.old:1]: @col-panic  It's an artefact, should be deleted. I will remove it in this PR.

File: docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js (hunk 1 above)
Comment (acomdigital) [reply docker/nextcloud/assets/apps/acom_custom_js_css/js/custom.js:280]: @col-panic Еxactly. It was integrated for develop. Could be deleted.

**Pull request merged**